calicoctl/calico_ctl/node.py

```diff
@@ -419,23 +370,31 @@ def _start_node_container_docker(ip, ip6, log_dir, node_image, detach, etcd_envs
         volumes=volumes)
     cid = container["Id"]
 
-    docker_client.start(container)
-    print "Calico node is running with id: %s" % cid
+    env_string = ""
+    for an_env in environment:
+        env_string += " -e " + an_env
 
-    if not detach:
-        _attach_and_stream(container)
+    vol_string = ""
+    for a_vol in binds:
+        vol_string += " -v %s:%s" % (a_vol, binds[a_vol]["bind"])
 
+    print "Running Docker container with the following command:\n"
+    print "docker run -d --restart=always --net=host --privileged --name=calico-node%s%s %s\n" % (env_string, vol_string, node_image)
```

Comment:
Will this command be right if the `--detach=false` option is passed?  It hardcodes `-d`.